### DISCUSSION TITLE
单独部署前端，后台的`自定义头部`和`自定义内容`失效

### DISCUSSION THREAD

**Discussion by youyou-sudo** [AlistGo/alist#4904] [Q&A] [answered]
## 我的操作
- 我使用 `git clone --recurse-submodules https://github.com/alist-org/alist-web.git` 拉取前端（也尝试不加 `--recurse-submodules`）
- 单独编译了前端，修改了 `.env.development`、 `vite.config.ts` 和 `.env.production` 中的 `api url` 为 `https://example.com/`
- 使用 `pnpm build` 编译出 `dist` 
- 部署到服务器后能正常访问列表数据，后台也能正常进入并更改设置

## 问题
后台中设置的`自定义头部`和`自定义内容`失效，样式变成了默认的

不知道是程序上设定的：“单独部署前端设置的`自定义头部`和`自定义内容`就失效”还是我打开很是有问题导致的
技术受限，脑袋空空(¦3[▓▓]，请求帮助！

## 目标
想让后台的`自定义头部`和`自定义内容`正常生效

**Comment by xhofe** [comment, marked as answer]
`自定义头部`和`自定义内容`实现的方法是 在后端程序中替换`index.html`后返回，单独部署无法替换。
